apis/core/v1beta1/openstackcontrolplane_types.go

```diff
@@ -171,7 +171,7 @@ type GaleraSection struct {
 	// +kubebuilder:validation:Optional
 	// +kubebuilder:default=false
```

Comment:
technically, we can keep omitempty for this one, as it defaulted to the bool type nil value?

Reply:
technically yes.